docs/core/diagnostics/dotnet-gcdump.md

```diff
@@ -164,11 +164,11 @@ dotnet-gcdump report [-h|--help] [-p|--process-id <pid>] [-t|--report-type <Heap
 
 - There is no type information in the gcdump.
 
-   Prior to .NET Core 3.1, there was an issue where a type cache was not cleared between gcdumps when they were invoked with EventPipe. This resulted in the events needed for determining type information not being sent for the second and subsequent gcdumps. This was fixed in .NET Core 3.1-preview2.
+   Prior to .NET 6, there was an issue where a type cache was not cleared between gcdumps when they were invoked with EventPipe. This resulted in the events needed for determining type information not being sent for the second and subsequent gcdumps. This was fixed in .NET 6-preview2.
```

Comment:
This is now wrong. This note doesn't feel relevant anymore.

Reply:
And same for the one below

Reply:
I'll delete the text since it is no longer needed